foundation_cms/static/scss/blocks/themes/default/two_column_container_block.scss

```
@@ -0,0 +1,6 @@
+.two-column-container {
+  padding: 2rem 0;
+  background: $white;
```

```suggestion
  background: map.get($mofo-colors, white);
```
because `$white` is referencing to a variable the legacy site uses and will be removed once we sunset the legacy code.

You will also need to include 

```scss
@use "sass:map";
@import "../../../colors";
``` 
at the top of this file for it to work I think.